### PR TITLE
Trained the model for avengers

### PR DESCRIPTION
**Pull request by Amit366**
#### Issue Number
ISSUE #9 

#### Describe the changes you've made
The model is trained to identify some of the avengers.

### CODE REVIEW

**Comment by Amit366** [comment]
@SanjayMarreddi sir please review it fast as only few days are left for the competition

File: Face_Recognition/Readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Face-Recognition
+
+This project is trained to identify the avengers like hulk, thor, captain_america and tony.
+
+The project asks the user for input and in the given photo if the above mentioned images are present then it represents the face with a rectangule with the name on it.
```
Comment (SanjayMarreddi) [inline Face_Recognition/Readme.md]: Do correct the `rectangle` typo.
Also You can briefly describe a bit about the Model here.

**Review by SanjayMarreddi — CHANGES_REQUESTED** [review]
@Amit366 It looks fine. Please make the slight modifications as suggested below :

- Make a Folder named `Basic_Avengers_Recognition`  under the `Facial_Recognition_Models/Using_Standard_Libraries` and then upload all your files there.
- Explain the difficult steps in the `script.py` file with inline Comments for better understanding.
-  Also briefly explain the Working of the Model & Classifications it can make in the README of newly created folder as said above.
- Make a Screen Recording while running the Model on your system & do keep a link of Video Demo in your README.
- You can refer to the README [here](https://github.com/SanjayMarreddi/Facial-Expression-Recognition-Classifier-Model/blob/master/Facial_Recognition_Models/Using_Standard_Libraries/Bharath_C_S/README.md) for an idea.

Good Luck !!

**Comment by Amit366** [comment]
@SanjayMarreddi sir video recording isn't possible as on running the script it asks the user to provide the image and from that image it shows whether the trained image is present or not. And the main problem is, dlib is showing error on my laptop so I asked my friend to run it on his laptop and it gave the desired results but now he went out of station.

**Comment by Amit366** [comment]
@SanjayMarreddi I will update the remaining changes

**Comment by Amit366** [comment]
@SanjayMarreddi I have updated it. Please review it.

**Review by SanjayMarreddi — CHANGES_REQUESTED** [review]
@Amit366 Now everything looks fine. But I asked you to put the `Basic_Avengers_Recognition` folder inside this `Facial-Expression-Recognition-Classifier-Model/Facial_Recognition_Models/Using_Standard_Libraries`.

Not under `Bharath_C_S`. Keep it as parallel to this.

Just make that modification.

**Comment by Amit366** [comment]
@SanjayMarreddi done sir. Please review it.

**Review by SanjayMarreddi — APPROVED** [review]

**Pull request merged**